### PR TITLE
Optimize [GithubIssues] using GraphQL

### PR DESCRIPTION
**Pull request by paulmelnikow**
While playing around with a badge for Hacktoberfest I noticed the GitHub Issues badge fetches a huge amount of JSON to render just the total count. There doesn’t seem to be a way to limit the response size through the REST API, so I thought I’d switch this to use GraphQL instead.

### CODE REVIEW

**Comment by calebcartwright** [comment]
Nice!

File: services/github/github-issues.service.js [hunk 1]
```diff
@@ -238,35 +255,98 @@ module.exports = class GithubIssues extends GithubAuthV3Service {
...
+        transformErrors,
+      })
+      return { issueCount: totalCount }
+    }
   }
 
   async handle({ variant, raw, user, repo, label }) {
-    const json = await this.fetch({ variant, user, repo, label })
+    const isPR = isPRVariant[variant]
+    const isClosed = isClosedVariant[variant]
```
Comment (paulmelnikow) [inline services/github/github-issues.service.js]: I also moved the logic of converting `variant` to `isPR` and `isClosed` to `handle()`. IMO as this concern is related to interpreting the badge URL, it doesn't really belong to fetching or rendering.

**Comment by calebcartwright** [comment]
Random question, but did you see/try https://developer.github.com/v4/explorer/ while working on this? 

I found it was a huge help for me while trying to figure out the queries for the Package Registry

File: services/github/github-issues.service.js (hunk 1 above)
Comment (calebcartwright) [reply services/github/github-issues.service.js]: Makes sense, this also does the lookup once instead too which is a plus 👍

**Comment by paulmelnikow** [comment]
> Random question, but did you see/try https://developer.github.com/v4/explorer/ while working on this?
> 
> I found it was a huge help for me while trying to figure out the queries for the Package Registry

Yea, it's super useful!

**Pull request merged**